### DISCUSSION TITLE
why is responsive removed on v7

### DISCUSSION THREAD

**Discussion by hieunm22** [katspaugh/wavesurfer.js#3487] [Q&A]
I'm using this lib to display audio file as replacement built-in html audio tag
Version `6.2.0` was running fine to me. but I had a trouble with test framework, so I had to update to version `^7.2.3`
I found that version 7 removed responsive props.
I want to know the reason of that removal. My wave component looks weird as below

<img width="379" alt="image" src="https://github.com/katspaugh/wavesurfer.js/assets/5255023/6ef27964-775e-4754-bd72-d0bb84abfd72">

and here's the wave component looks like in version 6

<img width="416" alt="image" src="https://github.com/katspaugh/wavesurfer.js/assets/5255023/92b45d4e-eee9-4337-a48d-899a617a56b0">

I don't know whether the changes to my component according to my package update is impacted or not
Could you tell me how to fix this issue? and tell me the reason why did you remove that props
Thanks

**Comment by katspaugh** [comment]
Duplicate of https://github.com/katspaugh/wavesurfer.js/discussions/3488
